app/controllers/display_ads_controller.rb

```diff
@@ -15,7 +15,7 @@ def show
         area: params[:placement_area],
         user_signed_in: user_signed_in?,
         user_id: current_user&.id,
-        article: ArticleDecorator.new(@article),
+        article: @article ? ArticleDecorator.new(@article) : nil,
```

Comment:
I'm just curious about this change, since I remember [this being introduced as a mechanism for dealing with missing article](https://github.com/forem/forem/pull/19285/commits/6e330c5bf358f5331ed95eb7f14bc0a66537d4c9) (eg, as opposed to the more succinct `@article&.decorate`, say).